solution_runner/commands/defaults_and_choices.py

```diff
@@ -0,0 +1,13 @@
+from datetime import datetime
+
+
+def get_default_year() -> int:
+    """
+    :return: default year which is the current year if it's December, last year otherwise
+    """
+    today = datetime.today()
+    current_year = today.year
+    if today.month == 12:
+        return current_year
+    else:
```

Comment:
else is not needed here, what does pep8 says about that?

Reply:
What do you mean by "not needed"? Do you suggest something like the following?
```python
    return current_year - (0 if today.month == consts.DECEMBER else 1)
    # Or:
    return current_year if today.month == consts.DECEMBER else current_year - 1

```

Reply:
Yes, or just remove the else.
```python
    If x:
        return x
    return y
```

Reply:
OK, I understand what you mean. Changed in d56f5fc3.